static/sass/styles.scss

```diff
@@ -1347,7 +1347,7 @@ $color-link-dark: #69c !default;
 .p-modal__dialog.is-paper {
   max-width: $grid-max-width !important;
 
-  &.is-ai-modal {
+  .row--50-50 {
```

Comment:
Are we applying this to all modals? Overriding an existing grid class could cause unintended effects, @lyubomir-popov what do you think?

Reply:
I made this change as it is a fix for when we use `row--50-50` in modals